src/stores/app.ts

```diff
@@ -25,6 +26,7 @@ export const useAppStore = defineStore('app', () => {
   const orgId = ref<string>(null)
   const developerSession = ref<SessionCookie>(null)
   const featuresetId = ref<string>(null)
+  const featureSet = ref<string>('')
```

Comment:
shouldn't it be `undefined` ?

Reply:
I think either undefined or '' would work here